b2b-app/imports/ui/admin/register/conditional-wrap.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const ConditionalWrap = ({ condition, wrapTrue, wrapFalse, children }) => {
```

Comment:
could we get a comment on what this function does? Its not immedietely clear to me by name